WORKSPACE

```diff
@@ -53,6 +53,9 @@ CLIKT_VERSION = "2.2.0"
 
 UI_AUTOMATOR_VERSION = "2.2.0"
 
+# NOTE: Artifacts are being pinned.  Whenever `maven_install` is changed, the
+# `maven_install.json` file should be updated. See the following page:
+# https://github.com/bazelbuild/rules_jvm_external#updating-maven_installjson
```

Comment:
Is there a command to generate the `manven_install.json` file? If so, it'd be nice to add it here.

Reply:
Done.